strawberryfields/gbs/similarity.py

```diff
@@ -270,7 +273,9 @@ def event_cardinality(photon_number: int, max_count_per_mode: int, modes: int) -
     return cardinality
 
 
-def prob_orbit_mc(graph: nx.Graph, orbit: list, n_mean: float = 5, samples: int = 1000) -> float:
+def prob_orbit_mc(
+    graph: nx.Graph, orbit: list, n_mean: float = 5, samples: int = 1000

```

Comment:
🤔 was this due to a run of `black`? Did we have the right line length before or after?

Reply:
I asked the same question over in #190 :laughing: